tests/testthat/test-qenv_get_code.R

```diff
@@ -632,14 +627,15 @@ testthat::test_that("detects cooccurrence properly even if all objects are on lh
   q <- eval_code(qenv(), code)
   testthat::expect_identical(
     get_code(q, names = "b"),
-    paste(code, collapse = "\n")
+    pasten(code)
   )
 })
 
 
 # @ ---------------------------------------------------------------------------------------------------------------
 
 testthat::test_that("understands @ usage and do not treat rhs of @ as objects (only lhs)", {
+  testthat::skip("This breaks on CI but not locally with: Error: cannot add bindings to a locked environment")
```

Comment:
Previously this was tested just on `code` that was a character and there was no evaluation.
Now we use `eval_code` and the code gets evaluated.

Reply:
Fixed in here 7cf13ac0b003be831c3d7dc0d9027dfecc99ba7a